src/components/CloudLogin/CloudLogin.tsx

```diff
@@ -82,7 +82,13 @@ export const CloudLogin: React.FC<CloudLoginProps> = ({
   };
 
   return (
-    <Flex direction="column" gap="2" maxWidth="540px" m="8px">
+    <Flex
+      direction="column"
+      gap="2"
+      maxWidth="540px"
+      m="8px"
```

Comment:
I think either `m="1"` or `m="2"`would be 8px * scaling

Reply:
This was already like that before, I feel like that could be its own pr since it is being done like that in more places